### ISSUE TITLE
Add user-agent to image-service to solve the image retrieval issue

### ISSUE THREAD

**Issue by sat01a** [AtlasOfLivingAustralia/image-service#178]
Museum Victoria and some of the other web services block downloading images from them as the image-service doesn't have a `User-Agent`.
After investigating the issue and with the Help of Simon and Dave, we realised that the `user-agent` needs to be specified in the image-service.
Simon tested the following script and it worked for MV:
```groovy
#!/bin/env groovy

def url = 'https://collections.museumsvictoria.com.au/content/media/6/1090906-large.jpg'.toURL()
def con = url.openConnection()
con.setRequestProperty('User-Agent', 'ALA-Image-Service/1.0')
println con.inputStream.getBytes().encodeHex()
```

**Comment by sat01a** [comment]
reported by @sadeghim

**Comment by sughics** [comment]
Released with [2.0.4](https://github.com/AtlasOfLivingAustralia/image-service/releases/tag/v2.0.4)
